mmv1/products/apigee/DnsZone.yaml

```
@@ -0,0 +1,104 @@
+# Copyright 2025 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'DnsZone'
+description: |
+  Apigee Dns Zone.
+references:
+  guides:
+    'Creating a DnsZone': 'https://cloud.google.com/apigee/docs/api-platform/get-started/create-dns'
+  api: 'https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.dnsZones/create'
+docs:
+base_url: 'dnsZones'
+self_link: '{{org_id}}/dnsZones/{{dns_zone_id}}'
+create_url: '{{org_id}}/dnsZones?dnsZoneId={{dns_zone_id}}'
+delete_url: '{{org_id}}/dnsZones/{{dns_zone_id}}'
+immutable: true
+import_format:
+  - '{{org_id}}/dnsZones/{{dns_zone_id}}'
+  - '{{org_id}}/{{dns_zone_id}}'
```

Adding % here would let org_id contain `/`, and would probably work. But you wouldn't get the same level of information in error messages if users specify it wrong.

```suggestion
  - '{{%org_id}}/dnsZones/{{dns_zone_id}}'
  - '{{%org_id}}/{{dns_zone_id}}'
```

---

Added the custom import code